app/src/main/java/com/example/b07project/ForgetPresenter.java

```diff
@@ -4,68 +4,55 @@ public class ForgetPresenter {
     private String email;
     LoginPresenter.PresenterViewPipe listener;
```

Comment:
yo u should probably delete this since that interface dont exist no more

Reply:
yes sir